lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -613,10 +614,7 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
     affected_routes = get_affected_routes(informed_entities)
     cause_text = get_cause_text(cause)
 
-    station =
-      case get_stations(informed_entities) do
-        [station | _] -> Stop.fetch_stop_name(station)
-      end
+    station = get_stations(informed_entities)
```

Comment:
Based on its name, `get_stations` returning a single string is a little confusing right now—maybe still have it return a list, rename the variable it's bound to to `bypassed_stations`, and call `Util.format_name_list_to_string` down below?

So—
```ex
bypassed_stations = get_stations(informed_entities)

%{
  issue: "Trains will bypass #{Util.format_name_list_to_string(bypassed_stations)}",
  ...
}
```